purchases/src/test/java/com/revenuecat/purchases/PurchasesTest.kt

```diff
@@ -2877,9 +2877,9 @@ class PurchasesTest {
 
         val lock = CountDownLatch(1)
         purchases.getCustomerInfoWith(onSuccess = {
-            lock.countDown()
-        }, onError = {
             fail("supposed to be successful")
```

Comment:
This should be `fail("supposed to be a failure")`

Reply:
😅  Good catch!

Reply:
![image](https://user-images.githubusercontent.com/3922667/174628194-a7c5006c-ebf8-457c-aa50-ce5577eae064.png)